src/app/chat/views/message_list.nim

```diff
@@ -246,6 +254,8 @@ QtObject:
     for c in contacts:
       for m in self.messages.mitems:
         if m.fromAuthor == c.id:
-          m.alias = userNameOrAlias(c)
+          m.userName = userNameOrAlias(c)
+          m.alias = c.alias
+          m.localName = c.localNickname
```

Comment:
Here's a little problem: it seems that `contact.alias` isn't deterministic. Sometimes it has the three word username alias, sometimes it holds the fully qualified ENS name, we should probably use `generateAlias()` from the contact's pubkey here instead...